pyworkflow/plugin.py

```diff
@@ -552,6 +551,22 @@ def validateInstallation(cls):
 
         return (["Missing variables:"] + missing) if missing else []
 
+    @staticmethod
+    def getTemplates(tempList, tDir, pluginName, templateFolder):
```

Comment:
I don't think we need any of those parameters, they should be inferred from inside.